client/blocks/app-promo/qr-code.jsx

```diff
@@ -0,0 +1,35 @@
+import { useLocalizeUrl } from '@automattic/i18n-utils';
+import { useTranslate } from 'i18n-calypso';
+import { QRCodeSVG } from 'qrcode.react';
+
+import './style.scss';
+
+export const QrCode = ( { source = 'calypso-customer-home', size = 150 } ) => {
```

Comment:
Similar to Mikaels feedback above, I'd make this required (ie remove the default value). Optionally, since this is a new component, why not make it in TypeScript?

Reply:
I updated the code to make it in TypeScript.